src/test/kotlin/no/nav/syfo/behandler/api/person/BehandlerPersonApiSpek.kt

```diff
@@ -0,0 +1,264 @@
+                            pid = UserConstants.ARBEIDSTAKER_FNR.value,
+                        )
+
+                        val fastlegeResponse = generateFastlegeResponse()
+                        with(
+                            handleRequest(HttpMethod.Get, url) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(validToken))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+
+                            val behandlerDialogmeldingList =
+                                objectMapper.readValue<List<BehandlerDialogmeldingPersonDTO>>(response.content!!)
+                            behandlerDialogmeldingList.size shouldBeEqualTo 1
+
+                            val behandlerDialogmeldingForPersonList = database.getBehandlerDialogmeldingForArbeidstaker(
+                                UserConstants.ARBEIDSTAKER_FNR,
+                            )
+                            behandlerDialogmeldingForPersonList.size shouldBeEqualTo 1
+
+                            val behandlerDialogmeldingDTO = behandlerDialogmeldingList.first()
+                            behandlerDialogmeldingDTO.fornavn shouldBeEqualTo fastlegeResponse.fornavn
+                            behandlerDialogmeldingDTO.mellomnavn shouldBeEqualTo fastlegeResponse.mellomnavn
+                            behandlerDialogmeldingDTO.etternavn shouldBeEqualTo fastlegeResponse.etternavn
+                            behandlerDialogmeldingDTO.adresse shouldBeEqualTo fastlegeResponse.fastlegekontor.postadresse?.adresse
+                            behandlerDialogmeldingDTO.postnummer shouldBeEqualTo fastlegeResponse.fastlegekontor.postadresse?.postnummer
+                            behandlerDialogmeldingDTO.poststed shouldBeEqualTo fastlegeResponse.fastlegekontor.postadresse?.poststed
+                            behandlerDialogmeldingDTO.telefon shouldBeEqualTo fastlegeResponse.fastlegekontor.telefon
+                            behandlerDialogmeldingDTO.orgnummer shouldBeEqualTo fastlegeResponse.fastlegekontor.orgnummer
+                            behandlerDialogmeldingDTO.kontor shouldBeEqualTo fastlegeResponse.fastlegekontor.navn
+                            behandlerDialogmeldingDTO.type shouldBeEqualTo BehandlerType.FASTLEGE.name
+                            behandlerDialogmeldingDTO.behandlerRef shouldBeEqualTo behandlerDialogmeldingForPersonList.first().behandlerRef.toString()
+                            behandlerDialogmeldingDTO.fnr shouldBeEqualTo fastlegeResponse.fnr
+
+                            database.getBehandlerDialogmeldingForArbeidstaker(
+                                UserConstants.ARBEIDSTAKER_FNR,
+                            ).size shouldBeEqualTo 1
+                        }
+                    }
+                    it("should return empty list of BehandlerDialogmelding for arbeidstaker uten fastlege") {
```

Comment:
Mye blanding av engelsk og norsk i disse `it`ene, også utenom domeneord; går det an å velge enten norsk eller engelsk?

Reply:
Ser det ble rot her ja, endret til bare engelsk nå.